content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -146,149 +146,56 @@ $ kubectl apply -f <(istioctl kube-inject \
      --filename $ORIGINAL_DEPLOYMENT_YAML)
 {{< /text >}}
 
-## Migrating per-service mutual TLS enablement via annotations to authentication policy
+### Migrating from `RbacConfig` to `ClusterRbacConfig`
 
-If you use service annotations to override global mutual TLS enablement for a service, you need to replace it with
-[authentication policy](/docs/concepts/security/#authentication-policies) and [destination rules](/docs/concepts/traffic-management/routing/destination-rules/).
+You can ignore the following if you have already upgraded to `ClusterRbacConfig`.
 
-For example, if you install Istio with mutual TLS enabled, and disable it for service `foo` using a service annotation like below:
+The `RbacConfig` is deprecated due to a [bug](https://github.com/istio/istio/issues/8825) in 1.1. You
```

Comment:
1.1 -> Istio 1.1

Reply:
Updated, actually the bug is in Istio 1.0